packages/paste-core/components/breadcrumb/stories/index.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Breadcrumb, BreadcrumbItem} from '../src';
+
+storiesOf('Components|Breadcrumb', module)
+  .addDecorator(withKnobs)
+  .add('Anchors', () => {
+    return (
+      <Breadcrumb>
+        <BreadcrumbItem>
+          <Anchor href="#">Phone Numbers</Anchor>
```

Comment:
Question: what if BreadcrumbItem was a wrapper around Anchor if a href="" was passed, otherwise it has the standard behavior.  So this could be:
```
<BreadcrumbItem href="#">Phone Numbers</BreadcrumbItem>
<BreadcrumbItem>My Phone Number</BreadcrumbItem>
```
This would also be a more future-proof way of controlling the styles of links in breadcrumbs and, if href is absent, we could automatically apply the ` aria-current="page"` to make sure it's present. Essentially this would be to try and have less user error.

Reply:
I had this same thought, but didn't wasn't sure if it was the right path to go down. Since you've mentioned it, and made me think about it more, this feels like a smart move. Less composing by the user, and would solve for the comment below about `Text`. I'll work this up.